### DISCUSSION TITLE
strapi-admin  final client bundle size is 6.2MB ,even compile with production mode!

### DISCUSSION THREAD

**Discussion by guanghw** [strapi/strapi#7290] [Legacy Topics]
strapi-admin  final client bundle size is 6.2MB ,even compile with production mode!Am I the only one who thinks the final compiled file is too big?

**Comment by yusufisl** [comment]
I think it is because the font awesome, i look the font awesome take almost the half of the size of bundle size

**Reply by derrickmehaffy** [reply]
I'll ping @soupette and @HichamELBSI to see if they have any suggestions on if we can decrease this by leveraging an outside resource that is possibly cachable or maybe a suggestion to swap out font-awesome for a native browser font that doesn't require outside resources.

**Reply by guanghw** [reply]
[https://github.com/FortAwesome/react-fontawesome/issues/70#issuecomment-411165298 ](url)
Is this the solution ?

**Comment by lauriejim** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

Then for this topic, I ping @soupette

**Reply by guanghw** [reply]
Ok!thanks!
